packages/core/src/module-resolver.ts

```diff
@@ -13,6 +13,7 @@ export function createDefaultResolver(fileSystem: MinimalFS, resolveOptions: any
         useSyncFileSystemCalls: true,
         cache: false,
         fileSystem,
+        extensions: ['.js', '.mjs', '.ts'],
```

Comment:
that doesn't seem quite right. `.mjs` is wrong. imports to such files in node must specify extensions. `.ts` is infrastructure specific (should come from a repository configuration).

Reply:
it's not for webpack. I actually saw that webpack default resolve options are not good for us they does not contain any extensions. also we don't use webpack infra to load the modules so we can't accept any configuration there.
the thing with `.mjs` is that it's not a node import it's Stylable I don't have a strong opinion to how we need to support them so no problem to remove it.
for `.ts` I don't have a problem saying that I can resolve them correctly by default. 

WDYT?

Reply:
remove .mjs.
.ts should come from a configuration, or by registering a custom resolver. we can't **assume** the user registers ts-node during his bundling, or otherwise.

Reply:
ok. accepted